StatsdClient/Statsd.cs

```diff
@@ -174,26 +203,27 @@ private void InitialiseInternal(Func<IOutputChannel> createOutputChannel, string
         /// <param name="period">The time period, can be one of h,d,dow,w,m</param>
         public async Task LogCalendargramAsync(string name, long value, string period)
         {
-            await SendMetricAsync(MetricType.CALENDARGRAM, name, _prefix, value, period);
+            await SendMetricAsync(MetricType.CALENDARGRAM, name, value, period);
         }
 
-        private async Task SendMetricAsync(string metricType, string name, string prefix, long value, string postFix = null)
```

Comment:
Originally I did that but it turns out that "postfix" has a different meaning in this context, so I decided to just move them both to PrepareMetric since they're both accessible to the class at that point.  Seemed slightly less ambiguous to me but that'st totally just my opinion.